test/replicate.js

```diff
@@ -1470,6 +1470,43 @@ test('can define default max-inflight blocks for replicator peers', async functi
   )
 })
 
+test('session id reuse does not stall', async function (t) {
+  t.plan(1)
+  const a = await create()
+  const b = await create(a.key)
+
+  const n = 5000
+
+  for (let i = 0; i < n; i++) await a.append(b4a.from([0]))
```

Comment:
make one batch instead, ie bill up an array with 5k blocks and pass that to array. much faster